protocols/cDNA_synth/cDNA_synth.ot2.py

```diff
@@ -70,7 +71,8 @@
 p50multi.transfer(1, samples.cols(), cdna_plate.cols(), new_tip='always')
 
 #  HOLD for 1.5hr for cDNA Synthesis
-p50single.delay(minutes=90)
+if not os.environ.get('OT_TESTING'):
```

Comment:
We can remove these `if not os.environ.get('OT_TESTING'):` checks for time.sleep / pipette.delay later when we have a protocol parser for OT2 & don't need to smoke test anymore.